### PR TITLE
feat: new home page copy

### PR DESCRIPTION
**Pull request by tomwisecodes**

### CODE REVIEW

File: apps/nextjs/src/app/home-page.tsx [hunk 1]
```diff
@@ -46,31 +45,50 @@ export default function HomePage({ featureFlag }) {
...
+            $display={["none", "flex"]}
+            $borderColor="black"
+            $borderStyle={"solid"}
+            $ba={"border-solid-l"}
+            customMaxWidth={600}
+            $height="fit-content"
+          >
+            <Image
+              src={"/images/aila-home-page-still.jpg"}
+              alt="jigsaw image"
```
Comment (mantagen) [inline apps/nextjs/src/app/home-page.tsx:85]: Not great alt text here

File: apps/nextjs/src/app/home-page.tsx [hunk 2]
```diff
@@ -46,31 +45,50 @@ export default function HomePage({ featureFlag }) {
...
+              width={700}
+              height={400}
```
Comment (mantagen) [inline apps/nextjs/src/app/home-page.tsx:87]: https://nextjs.org/docs/app/building-your-application/optimizing/images#local-images

Looks like width/height get calculated automatically

**Review by mantagen — COMMENTED** [review]
Def not a blocker but could have double size image for high density screens

**Review by codeincontext — APPROVED** [review]

File: apps/nextjs/src/app/home-page.tsx (hunk 2 above)
Comment (tomwisecodes) [reply apps/nextjs/src/app/home-page.tsx:87]: throws an error if you dont put them in

Comment (mantagen) [reply apps/nextjs/src/app/home-page.tsx:87]: Gotta do like

```
import profilePic from './me.png'

    <Image
      src={profilePic}
```

but no worries

**Review by mantagen — APPROVED** [review]

**Pull request merged**